src/jobs/NotificationTrigger/index.js

```diff
@@ -19,37 +19,51 @@ module.exports = async function(context) {
 					currentTime,
 				}
 			})
+			if (userWithCurrentTime) {
+				const article = await newsDbService.getLatestNewsArticle()
+				const todaysTimeFrame = getStartEndTime()

```

Comment:
@chetannn isn't start time going to be user-specific? this function seems generic.
Are all times in db in GMT?